src/io/mod.rs

```diff
@@ -1,8 +1,12 @@
 //! Readers and writers for common bioinformatics file formats.
 
 pub mod bed;
+pub mod bnx;
+pub mod cmap;
 pub mod fasta;
 pub mod fastq;
 pub mod gff;
 #[cfg(feature = "phylogeny")]
 pub mod newick;
+pub mod om_utils;
+pub mod xmap;
```

Comment:
Rather have a submodule `om` and in there bnx, cmap, xmap, and put the stuff of om_utils into a module om::common.